### ISSUE TITLE
Midterm Peer review-hw672

### ISSUE THREAD

**Issue by Howard233** [aliciayuting/ORIE4741-project#5]
Summary:
The project is trying to detect default credit card clients as accurately as possible using machine learning algorithms. The dataset the team is using is the credit card holders’ registration information and their transaction data (from April to September, 2015). They constructed a classification problem and will use supervised machine learning model such as perceptron. 

Three things I like:
The project gives detailed description on data source and dataset itself. It tells me the features in the dataset and what does each feature mean. The data is also very big and messy which reflects the topic of the course. Another thing I like is that they included many plots, including histogram, pie chart and box plot. All of these visualizations help me understand the data better. I also like their detailed introduction to the error metrics. They will use confusion matrix, recall accuracy and ROC to test their model effectiveness. 

Three things I don’t like:
The project tells that the default customers have approximately proportion of 22.216%, and they tried to split the data into 3 parts: training set, valuation set and test set. I think the dataset is very imbalanced and they should explain how they will handle this problem. Also, the project tells that the team will use random forest, neural network and XGBoost. I wonder if they could try different classification methods such as logistic regression and KNN. I would also like to see their explanation on how to do feature engineering which is very important on model building.
